### PR TITLE
A few changes to make it neater and a more useful invite command

### PR DESCRIPTION
**Pull request by sgmedev**

### CODE REVIEW

File: common.py [hunk 1]
```diff
@@ -9,262 +9,301 @@
...
-    await ctx.send(f"To donate to support the development and hosting of {self.bot.user.mention}, you can donate via Patreon!\nhttps://www.patreon.com/Gnome_the_Bot_Maker")
-
-  @commands.command()
-  @commands.bot_has_permissions(send_messages=True, read_messages=True)
-  async def botstats(self, ctx):
-    message = cleandoc(f"""
+    def __init__(self, bot):
+        self.bot = bot
+
+    @staticmethod
```
Comment (sgmedev) [inline common.py:44]: I'm not sure what `@staticmethod` does but Pycharm said to add it

I think it probably means that it doesn't use `self`

File: common.py [hunk 2]
```diff
@@ -9,262 +9,301 @@
...
+    # F@H
+    "565820959089754119": "https://discordapp.com/api/oauth2/authorize?client_id=565820959089754119&permissions=84992&scope=bot",
+    # Channel maker
+    "689564772512825363": "https://discordapp.com/api/oauth2/authorize?client_id=689564772512825363&permissions=68624&scope=bot",
+
+    # TTS bot dev for testing
+    "698218518335848538": "https://example.com"
+}
+
+colours = {
```
Comment (sgmedev) [inline common.py:24]: don't complain about my spelling its spelt correctly I don't care about the other more "correct" way to spell it

File: common.py [hunk 3]
```diff
@@ -9,262 +9,301 @@
...
+        if invite is None:
+            await ctx.send("No channel to make invite for or no perms")
+        else:
+            await ctx.send(str(invite))
+
+    @commands.command()
+    @commands.is_owner()
+    async def sudo(self, ctx, user: typing.Union[discord.Member, discord.User, str], *, message):
+        """mimics another user"""
+        if isinstance(user, str):
```
Comment (sgmedev) [inline common.py:96]: I think this is a better way to do what you're trying to do this is what Pycharm said to do instead

**Comment by sgmedev** [comment]
Someone should probably make a lot of the ids variables so that it can be configured more easily or even out the config in a separate file

**Comment by GnomedDev** [comment]
There is a reason -invite works by inviting to the support server, it is to grow it and to also let people stay for updates and other stuff. I am ok with dadbd72 and the -sudo improvements in 53fe002, but other than that don't want -invite fundamentally changed, maybe make it link to the bot invite in the support server? (check if self.bot.supportserver, `if guild.id == self.bot.supportserver.id`)

**Comment by sgmedev** [comment]
If people want to hear about updates they will join your server but some people dont really care about updates in myy oppinion it is much better to be able to invite the bot from -invite as that neans that people can add the bot more easilly

**Comment by GnomedDev** [comment]
I have had this conversation multiple times before with different people, I think I have even told you. -invite isn't going to link the bot invite.

**Comment by sgmedev** [comment]
There must be a reason why almost every other bot in existance does this can you at ,east do a poll on discord?

**Comment by GnomedDev** [comment]
Possibly, if it becomes such an issue. (also kinda laughing at you thumbs upping your own messages)

**Comment by sgmedev** [comment]
Is it posseble to merge my other two commits for now and leave this pr open until we finnish our debate?

**Comment by GnomedDev** [comment]
Implemented what I wanted in 27ef918 and 34fe0ad, closing

**Pull request closed without merging**